rally-domain/src/integration-test/groovy/yakworks/rally/orgs/OrgRepoTests.groovy

```diff
@@ -340,11 +340,15 @@ class OrgRepoTests extends Specification implements DomainIntTest {
     }
 
     void "test find org by sourceid"() {
```

Comment:
name "test lookup by sourceid", add lookup to other test name to so it more obvious